src/app/domain/geolocation/helpers/webhook.ts

```diff
@@ -0,0 +1,31 @@
+import CozyClient from 'cozy-client'
+
+import { buildServiceWebhookQuery } from '/app/domain/geolocation/helpers/index'
+import { storeFetchServiceWebHook } from '/app/domain/geolocation/tracking'
+import { FETCH_OPENPATH_TRIPS_SERVICE_NAME } from '/app/domain/geolocation/tracking/consts'
+
+interface WebhookTrigger {
+  message: {
+    name: string
+  }
+  links?: {
+    webhook?: string
+  }
+}
+
+export const fetchWebhook = async (client: CozyClient): Promise<void> => {
+  const webhookQuery = buildServiceWebhookQuery()
+  const webhookResp = await client.fetchQueryAndGetFromState(webhookQuery)
```

Comment:
Why using `fetchQueryAndGetFromState`  instead of `useQuery`?

Reply:
We can not use useQuery here because we are not in a hook or a component. And even without the refactoring, we can not use a hook after a condition (basically I do `if client then query`).

Reply:
And you can't use client.query because of https://github.com/cozy/cozy-client/issues/931 & https://github.com/cozy/cozy-client/issues/867

With all the links to the issue, I think we should work on it ^^'